backend/modules/auth.ts

```diff
@@ -1,7 +1,7 @@
 import { type NextFunction, type Request, type Response } from 'express'
 
 export const checkUserAuthorization = (req: Request, res: Response, next: NextFunction): void => {
-  if (req.user === null) {
+  if (!req.user) {
```

Comment:
we can revert this back after the explicit check rule is reverted

Reply:
done!